### PR TITLE
user permissions instead of querying groups

### PR DESCRIPTION
**Pull request by das-g**

### CODE REVIEW

File: osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py [hunk 1]
```diff
@@ -8,7 +8,13 @@
...
     """Add group osmaxx."""
     Group = apps.get_model("auth", "Group")
-    Group.objects.create(name=settings.OSMAXX_FRONTEND_USER_GROUP)
+    Permission = apps.get_model("auth", "Permission")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    ExtractionOrder = apps.get_model("excerptexport", "ExtractionOrder")
+    group = Group.objects.create(name=settings.OSMAXX_FRONTEND_USER_GROUP)
+    content_type = ContentType.objects.get_for_model(ExtractionOrder)
+    permission, created = Permission.objects.get_or_create(codename='add_extractionorder', content_type=content_type)
+    group.permissions.add(permission)
```
Comment (das-g) [inline osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py]: Now that rest of `feature/#12_provide-initial-access-groups-and-permissions` got already merged in #143, should the adding of the relationship from the group to the permission be in a separate migration?

File: osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py [hunk 2]
```diff
@@ -8,7 +8,13 @@
 def update_site_forward(apps, schema_editor):
     """Add group osmaxx."""
     Group = apps.get_model("auth", "Group")
-    Group.objects.create(name=settings.OSMAXX_FRONTEND_USER_GROUP)
+    Permission = apps.get_model("auth", "Permission")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    ExtractionOrder = apps.get_model("excerptexport", "ExtractionOrder")
+    group = Group.objects.create(name=settings.OSMAXX_FRONTEND_USER_GROUP)
+    content_type = ContentType.objects.get_for_model(ExtractionOrder)
```
Comment (das-g) [inline osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py]: As we get **or create** a permission, should we remove it again in the backwards migration? (Probably not, as it'd be recreated by the `post_migrate` signal handler, anyway.)

File: osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py (hunk 1 above)
Comment (njordan-hsr) [inline osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py]: Would be an argument for it, but I think this is just fine: backwards migrations would be much harder when having separate migrations.

File: osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py (hunk 2 above)
Comment (njordan-hsr) [inline osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py]: That is correct, AFAIK we only need to remove the group.

**Pull request merged**